fuel/app/migrations/047_add_score_unscorable_to_log.php

```diff
@@ -0,0 +1,24 @@
+<?php
+namespace Fuel\Migrations;
+class Add_score_unscorable_to_log
+{
+	public function up()
+	{
+		\DBUtil::modify_fields(
+			'log',
+			[
+				'type' => ['type' => 'enum', 'constraint' => "'BUTTON_PRESS', 'ERROR_GENERAL', 'ERROR_TIME_VALIDATION', 'KEY_PRESS', 'SCORE_ACTIVITY_FROM_CLIENT', 'SCORE_FINAL_FROM_CLIENT', 'SCORE_QUESTION_ANSWERED', 'SCORE_WIDGET_INTERACTION', 'SCORE_UNSCORABLE', 'WIDGET_CORE_INIT', 'WIDGET_END', 'WIDGET_LOAD_DONE', 'WIDGET_LOAD_START', 'WIDGET_LOGIN', 'WIDGET_PLAY_REQ', 'WIDGET_PLAY_START', 'WIDGET_RESTART', 'WIDGET_START', 'WIDGET_STATE', 'DATA'", 'null' => true]
+			]
+    );
+  }
+
+	public function down()
+	{
```

Comment:
This throws an SQL error, `Uncaught exception PDOException: 01000 - SQLSTATE[01000]: Warning: 1265 Data truncated for column 'type' at row...`

If there are any rows in the `log` table where the value is `SCORE_UNSCORABLE`, the modify query will error out since it's changing which values are allowed to exist in that column while there are rows in the table with a value that would no longer be allowed.

In order to avoid this, you'd have to make sure you remove any rows in the table with the invalid value OR modify them to have a valid value - either way, you'd have to do that prior to the modify query.

For example, to delete the would-be-erroneous rows:
```suggestion
\DB::delete('log')->where('type', 'SCORE_UNSCORABLE')->execute();
```